content/en/network_monitoring/performance/setup.md

```diff
@@ -344,6 +344,44 @@ If you already have the [Agent running with a manifest][4]:
 [3]: https://app.datadoghq.com/organization-settings/api-keys
 [4]: /agent/kubernetes/
 {{% /tab %}}
+{{% tab "Operator" %}}
+[The Datadog Operator][1] is a way to deploy the Datadog Agent on Kubernetes and OpenShift. It reports deployment status, health, and errors in its Custom Resource status, and it limits the risk of misconfiguration thanks to higher-level configuration options.
```

Comment:
This should include the same beta warning as https://docs.datadoghq.com/agent/kubernetes/?tab=operator#deploy-an-agent-with-the-operator